test/service/test_dss_proxy.py

```diff
@@ -106,7 +105,7 @@ def test_dss_files_proxy(self):
                                                           url=dss_url.url,
                                                           status=301,
                                                           headers={'Location': dss_url_with_token.url,
-                                                                   'Retry-After': str(retry_after)}))
+                                                                   'Retry-After': str(10)}))  # Default value from DSS
```

Comment:
```suggestion
                                                                   'Retry-After': str(10)}))
```

What the DSS actually returns might change in the future, in which case the comment becomes false.